src/components/Proposal/Proposal.scss

```diff
@@ -324,11 +324,22 @@ button.disabled {
     float: right;
     border: none;
     margin-top: 13px;
-    border: none;
     height: 20px;
     line-height: 20px;
 }
 
+.detailView .proposalInfo .voteStatus.hasVoted .voteRecord {
+	display: inline-block;
```

Comment:
there is weird indentation all throughout the CSS, make sure you dont have tabs turned on

Reply:
OK. I did restructure the CSS a bit, but I think Sublime just uses whatever scheme is present. "detect_indentation": true...